backend/certificates.js

```diff
@@ -34,6 +34,34 @@ export function saveCertsLocally(json) {
     saveSetting('CONNECTOR_HOST_INFO', hostInfo);
     return {};
 }
+
+export function fetchCertsFromCA() {
+    // TODO: What to do when more than one user?
+    // TODO: Should I try getting the api_key if no access_token?
```

Comment:
probably not necessary anymore. I put in the `api_key` settings in case users wanted to configure it themselves in their settings file like on a server, but it's not really necessary anymore with the oauth flow